lib/loaders/express.loader.ts

```diff
@@ -48,6 +48,14 @@ export class ExpressLoader extends AbstractLoader {
         app.use(express.static(clientPath, options.serveStaticOptions));
         app.get(options.renderPath, renderFn);
       }
+
+      app.use((err: Error, req, res, next: Function) => {
+        if (err.message != undefined && err.message.includes('ENOENT')) {
+          res.setStatus(HttpStatus.NOT_FOUND);
```

Comment:
```suggestion
          return res.send(new NotFoundException(err.message));
```

`.setStatus` doesn't exist, it's named `.status` but it's anyways not enough.

My suggested changes result in that

```json
{
  "response": {
    "statusCode": 404,
	"message": "ENOENT: no such file or directory, stat '/usr/local/foo/bar/index.html'",
	"error": "Not Found"
  },
  "status": 404,
  "message": "ENOENT: no such file or directory, stat '/usr/local/foo/bar/index.html'"
}
```